lib/tasks/check_changelog.rake

```diff
@@ -17,8 +17,13 @@
 require 'rake'
 
 namespace "check" do
-  desc "Checking for new IDs (bugzilla,fate,...)  in *.changes file"
+  desc "Checking for new IDs (bugzilla,fate,...) in *.changes file"
   task :changelog do
+    if obs_sr_project.nil?
+      $stderr.puts "Skipping the check because OBS submit request project "\
```

Comment:
NP: `Skipping the changelog check ...` would be better as it is included in another checks, just to be more clear.